src/main/scala/org/broadinstitute/dsp/HelmAlgebra.scala

```diff
@@ -9,7 +9,8 @@ import scala.util.control.NoStackTrace
 trait HelmAlgebra[F[_]] {
   def installChart(
     release: Release,
-    chart: Chart,
+    chartName: ChartName,
+    chartVersion: ChartVersion,
```

Comment:
Was wondering if we should make the version default to latest if not specified? I suppose it makes sense to enforce always setting the version in the library if that's what Leo is going to do.

Reply:
Yea, I thought about the same thing and decided to be explicit given the use case today. If that changes, we can modify the API.